### PR TITLE
add 'autosubmit' to the dependabot config

### PR DESCRIPTION
**Pull request by devoncarew**
- add 'autosubmit' to the dependabot config

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

#### Contribution guidelines:

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Many Dart repos have a weekly cadence for reviewing PRs - please allow for a week or two of latency for initial review feedback.

---

### CODE REVIEW

File: tool/ci.sh [hunk 1]
```diff
@@ -67,10 +67,18 @@ for PKG in ${PKGS}; do
       echo
       echo -e "\033[1mPKG: ${PKG}; TASK: ${TASK}\033[22m"
       case ${TASK} in
-      analyze)
+      analyze_0)
         echo 'dart analyze --fatal-infos .'
```
[the remark this answers is not in the document]
Comment (kevmoo) [reply tool/ci.sh:71]: did you eliminate mono_repo?

**Review by kevmoo — APPROVED** [review]

File: tool/ci.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (devoncarew) [reply tool/ci.sh:71]: This is re-generating the mono repo files -

[the remark this answers is not in the document]
Comment (kevmoo) [reply tool/ci.sh:71]: When I run `mono_repo generate` locally things are...different. Did you hand-modify things?

[the remark this answers is not in the document]
Comment (devoncarew) [reply tool/ci.sh:71]: No - that's odd. And I think I was running w/ the latest mono_repo.

[the remark this answers is not in the document]
Comment (kevmoo) [reply tool/ci.sh:71]: Did a `mono_pkg` file get deleted? Or ignored or something?

**Pull request merged**